tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/lock/LocksWithOnFailTest.java

```diff
@@ -46,8 +46,7 @@ public void testLockWithinLock() {
         assertSame(returnsWithFail.get(0).getClass(), Long.class);
         assertTrue(returnsWithFail.get(1) instanceof BString);
 
-        assertEquals(returnsWithFail.get(0), 100L);
-        assertEquals(returnsWithFail.get(1).toString(), "Error caught");
+        BRunUtil.invoke(compileResult, "onFailLockWithinLockWithoutVariable");
```

Comment:
Let's make this a separate test. We shouldn't call multiple b7a test functions from a Java test. If one fails the rest won't run.